eng/scripts/Publish-Crates.ps1

```diff
@@ -14,8 +14,8 @@ foreach ($crateName in $CrateNames) {
   Write-Host "Publishing packae: '$crateName'"
   $manifestPath = "$PackagesPath/$crateName/Cargo.toml"
   # https://doc.rust-lang.org/cargo/reference/registry-web-api.html#publish
-  Write-Host "> cargo publish --locked --manifest-path `"$manifestPath`""
-  cargo publish --locked --manifest-path $manifestPath
+  Write-Host "> cargo publish --manifest-path `"$manifestPath`""
+  cargo publish --manifest-path $manifestPath
```

Comment:
In case it matters, these commands aren't the same. You quote `$manifestPath` in one but not the other. Best to be consistent in case it ever does matter and we're confused one way or the other. I think it's safe *not* to quote.

Reply:
Powershell doesn't need quotes to pass the arg to cargo publish, but if someone doesn't have the string already in a variable, they may need quotes to make the same call.

I want someone to be able to copy paste the `>` line from the build log to repro an error, so I quote the string variables, but only in the "copy paste this to repro" log line